proto/profile/tx.proto

```diff
@@ -32,6 +33,16 @@ message MsgDeleteValidator {
 
 message MsgDeleteValidatorResponse {}
 
+message MsgSetValidatorConsAddress {
+  string validatorAddress = 1;
+  bytes validatorPubKey = 2;
```

Comment:
```suggestion
  bytes validatorConsPubKey = 2;
```

We should precise this is the consensus key which is different from the validator SDK address